diesel_cli/src/main.rs

```diff
@@ -606,6 +607,31 @@ fn regenerate_schema_if_file_specified(
     Ok(())
 }
 
+// List which features have been compiled into this version of diesel_cli
+#[cfg(any(feature = "postgres", feature = "mysql", feature = "sqlite"))]
```

Comment:
Don't need this line.

Reply:
I will remove `#[cfg(any(feature = "postgres", feature = "mysql", feature = "sqlite"))]`